Unity-Technologies/ScriptableRenderPipeline
# Added a fp16 maximum epsilon definition

### Purpose of this PR
FLT_EPS used with fp16 was generating glitches because of Nans. We added a definition for REAL_EPS that becomes HALF_EPS when fp16 are used.

---
### Release Notes
When using a hlsl function with real, we use REAL_EPS.

---
### Testing status
**Katana Tests**: running

**Manual Tests**: What did you do?
- Opened sample scene on PC and check every thing was ok
- Ran iOS level with half float use activated.


Launch Katana (Link on master here - update for your branch):
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=add-fp16-maximum-precision&automation-tools_branch=master&unity_branch=trunk


---
### Overall Product Risks
**Technical Risk**: None

**Halo Effect**: None

---
### Comments to reviewers
Notes for the reviewers you have assigned.
